Modules/_uuidmodule.c

```diff
@@ -0,0 +1,40 @@
+#define PY_SSIZE_T_CLEAN
+
+#include "Python.h"
+#include <uuid/uuid.h>
+
+
+static PyObject *
+_uuid_generate_time_safe(void)
```

Comment:
I suggest to add a py prefix: py_uuid_generate_time_safe().

Reply:
Agreed.